### ISSUE TITLE
Почему ты еще в России?

### ISSUE THREAD

**Issue by Blazelip** [smartvotedocs/votesmart2021#27]
Каждый второй айтишник хочет свалить из страны и может сделать это, но почему-то не делает. 
Я понимаю, что при жизни в СНГ и айтишной зп, у нас намного больше остается, чем у среднего "Россиянина", то есть уровень жизни выше, чего в Европе и в других развитых странах не наблюдается. Но тем не менее, безопасность "жепы" на эти деньги не купить.
Так вот, почему ты еще тут?
---------------------------------------------------------------------
Пилю ютубчик на досуге, чекни https://www.youtube.com/c/Filyukov

**Comment by vshuraeff** [comment]
отвечу вопросом на вопрос: почему всех постоянно прогоняют из России?

**Comment by Blazelip** [comment]
> отвечу вопросом на вопрос: почему всех постоянно прогоняют из России?

Цели необоснованно перднуть "ВАЛИ ИЗ РАШКИ", у меня не стояло под этим тредом, я люблю свою Родину, но к сожалению, личная безопасность и возможность открывать рот, кмк, дороже любых денег.

**Comment by YuriyMironenko** [comment]
Мне кажется, переезд не такая простая штука, как хотелось бы. Язык учи, визу получи, семью перевези, на месте адаптируйся. Т.е. нужно быть готовым к серьёзной потере в качестве жизни на несколько лет с неясными перспективами.

**Comment by Blazelip** [comment]
> Мне кажется, переезд не такая простая штука, как хотелось бы. Язык учи, визу получи, семью перевези, на месте адаптируйся. Т.е. нужно быть готовым к серьёзной потере в качестве жизни на несколько лет с неясными перспективами.

Да, ты чертовски прав, вначале будет несладко, и захочется вернуться

**Comment by tastyreefer** [comment]
> > отвечу вопросом на вопрос: почему всех постоянно прогоняют из России?
> 
> Цели необоснованно перднуть "ВАЛИ ИЗ РАШКИ", у меня не стояло под этим тредом, я люблю свою Родину, но к сожалению, личная безопасность и возможность открывать рот, кмк, дороже любых денег.

Не раззевай рот, ворона залетит))))

**Comment by WilhelmNeumann** [comment]
> > Мне кажется, переезд не такая простая штука, как хотелось бы. Язык учи, визу получи, семью перевези, на месте адаптируйся. Т.е. нужно быть готовым к серьёзной потере в качестве жизни на несколько лет с неясными перспективами.
> 
> Да, ты чертовски прав, вначале будет несладко, и захочется вернуться

Я уехал и это не так сложно, как может показаться

**Comment by Yogendrovich** [comment]
Потому, что у меня уже нет иллюзий по поводу среды для жизни в других странах. Для меня важнее жить в месте среди людей, с которыми я хотя-бы могу общаться на языке, где моя речь не будет слишком искусственной и среди общества где я понимаю юмор и шутки. Это куда важнее чем открывать рот на политические и абстрактные общественные темы.

**Comment by gmankab** [comment]
мне тут нравится, несмотря на отстойнейшую власть тут очень приятные люди.

**Comment by procion1302** [comment]
Потому, что практика показала, что здесь я нужнее.
Тут для посредственного погромиста всегда найдется работа, а вот в моей желанной стране требования предъявили куда выше.
